web/src/popover_menus.js

```diff
@@ -190,6 +190,24 @@ function on_show_prep(instance) {
     popovers.hide_all_except_sidebars();
 }
 
+// This function is created to instantly toggle the popover menu instead of
+// registering events on `body`. This is useful for cases when we,
+// for example, trigger the popover by a keyboard shortcut.
```

Comment:
Nit: "This function is created" is something you never want to put in a code comment, because while it's a very reasonable way to talk about a change being made, someone who is reading the code as it exists in the codebase later just wants to read an explanation of the code itself, not its change history, which is invisible to them.

Reply:
I'll rewrite this comment since I think it's the only blocker for merging this commit.

Reply:
Thanks, Tim!